traffic_monitor/tests/gocover.bash

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#----------------------------------------
+set -e
+go get "github.com/wadey/gocovmerge"
```

Comment:
You should be fetching dependencies that exist for the image's entire run-time during the build step.

But also, has this dependency been approved on a license basis?

Reply:
I might need some enlightenment here

Reply:
You can ask the mailing list about using this library in this way.